contributed_definitions/NXoptical_spectroscopy.nxdl.xml

```diff
@@ -693,7 +700,7 @@ TODO:
                     <enumeration>
                         <item value="long-pass filter"/>
                         <item value="short-pass filter"/>
-                        <item value="Notch filter"/>
+                        <item value="notch filter"/>
```

Comment:
see comments above, open the yaml file and search for enumeration: then you can easily find these situations

Reply:
resolved as underscores are not mandatory